exercises/change/cases_test.go

```diff
@@ -0,0 +1,138 @@
+package change
+
+// Source: exercism/x-common
+// Commit: cda8f98 Create new exercises structure
```

Comment:
Hang on, this is my fault, I ran it against an out-of-date x-common. I've rebased and the example_gen.go works fine, sorry! 

It would be good to re-generate the test cases here before merging.